src/probnum/filtsmooth/gaussian/approx/_interface.py

```diff
@@ -0,0 +1,115 @@
+"""Temporary interface."""
```

Comment:
The class in here used to be EKFComponent. I am planning on kicking it out sooner rather than later, but for the time being, it is useful to make sure that the UKF and the EKF have very similar API (which will make refactoring easier, later)

Reply:
it is not introduced but rebranded, if that makes sense. The kicker is that it can be reused for the current UKF implementation, so it saves a lot of duplicate code. Maybe the module docstring is misleading?

Reply:
I see. Mh, maybe a very short docstring stating exact that (temporary, etc.) couldn't harm? 😄 
Thanks!

Reply:
```suggestion
"""Temporary interface. Will eventually be removed as a part of the refactoring detailed in issue #627."""
```